ignite/contrib/handlers/neptune_logger.py

```diff
@@ -288,6 +288,22 @@ def global_step_transform(*args, **kwargs):
                 global_step_transform=global_step_transform
             )
 
+        Another example where the State Attributes ``trainer.state.alpha`` and ``trainer.state.beta``
+        are also logged along with the NLL and Accuracy after each iteration:
+
+        .. code-block:: python
+
+            npt_logger.attach(
+                trainer,
+                log_handler=OutputHandler(
+                    tag="training",
+                    metric_names=["nll", "accuracy"],
+                    state_attributes=["alpha", "beta"],
+                ),
+                event_name=Events.ITERATION_COMPLETED
```

Comment:
@Ishan-Kumar2 let's update here the docstring as well: attach -> attach_output_handler